sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/CosmosConstants.scala

```diff
@@ -18,7 +18,8 @@ private object CosmosConstants {
   val initialMaxRetryIntervalForTransientFailuresInMs = 100
   val maxRetryIntervalForTransientFailuresInMs = 5000
   val maxRetryCountForTransientFailures = 100
-  val defaultDirectRequestTimeoutInSeconds = 10L
+  val defaultDirectRequestTimeoutInSeconds = 20L
```

Comment:
Looks like there is a test regression because of this change (direct connectivity mode max request timeout is at 10s).

Reply:
Fixed